pkg/appgw/frontend_listeners_test.go

```diff
@@ -145,10 +142,9 @@ var _ = Describe("MutateAppGateway ingress rules and parse frontend listener con
 				FrontendIPConfiguration:     resourceRef(tests.PublicIPID),
 				FrontendPort:                resourceRef(resPref + "frontendPorts/fp-443"),
 				Protocol:                    n.ApplicationGatewayProtocol("Https"),
-				HostName:                    to.StringPtr(tests.Host),
 				SslCertificate:              resourceRef(resPref + "sslCertificates/--namespace-----the-name-of-the-secret--"),
 				RequireServerNameIndication: to.BoolPtr(false),
-				Hostnames:                   &[]string{},
+				Hostnames:                   &[]string{tests.Host},
```

Comment:
we might need to cover annotation hostname-extension for your code change?

Reply:
we already have tests this annotation.